src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString.split(';');
+  const cssStyles = {};
+
+  styles
+    .filter((style) => style.trim())
+    .forEach((style) => {
+      const parts = style.split(':');
+      const styleKey = parts[0];
+      const styleValue = parts[1];
+
+      cssStyles[styleKey.trim()] = styleValue.trim();
+    });
```

Comment:
Everything is good, but chaining contributes to the worse readability of the code. Try splitting the code into constants.
Also, you can use the reduce method, so you don't need to create cssStyles object.